File: Day2/PythonPrograms/NumberCheck.py
def check_number(num):
    if num > 0:
        print(f"{num} is positive")
    elif num < 0:
        print(f"{num} is negative")
    else:
        print(f"{num} is zero")

# Test the function
check_number(5)   # 5 is positive
check_number(-3)  # -3 is negative
check_number(0)   # 0 is zero

File: Day2/javaPrograms/NumberCheck/NumberCheck.java
package NumberCheck;

public class NumberCheck {
    public static void main(String[] args) {
        checkNumber(5); // 5 is positive
        checkNumber(-3); // -3 is negative
        checkNumber(0); // 0 is zero
    }

    public static void checkNumber(int num) {
        if (num > 0) {
            System.out.println(num + " is positive");
        } else if (num < 0) {
            System.out.println(num + " is negative");
        } else {
            System.out.println(num + " is zero");
        }
    }
}
